### ISSUE TITLE
Swap out usgs analytics code 

### ISSUE THREAD

**Issue by jread-usgs** [USGS-CIDA/OWI_website#38]
```html
</script>
<script type="application/javascript" src="http://www.usgs.gov/scripts/analytics/usgs-analytics.js"></script>
</head>
```
That is really slow. @jkreft-usgs says https://www2.usgs.gov/scripts/analytics/usgs-analytics.js is not as slow

**Comment by jkreft-usgs** [comment]
It should be LESS slow.  I'm not saying that it won't be slow, because it is still a synchronous request off a underperforming server.  It should just be better than going through three redirects like we have to right now.

**Comment by jread-usgs** [comment]
yeah ☝️, not as slow

**Comment by jkreft-usgs** [comment]
fixed with #46
